climpred/classes.py

```diff
@@ -91,6 +91,16 @@ def __init__(self, xobj):
     def __repr__(self):
         return _display_metadata(self)
 
+    def smooth(self):
```

Comment:
just tried to implement it. smooth_dict would need to change from `init` to `time`. I would like to try to implement this into the main class and not the subclass, because then we will have to do things twice. I actually wanted to loop over all objects in self. didnt work.